lib/mri/syslog/logger.rb

```diff
@@ -0,0 +1,209 @@
+# frozen_string_literal: false
```

Comment:
This file is from MRI 2.3.3, right?
Can you adapt `tool/update-mri.sh` to include this file and the .c and make sure it works?